moderation.go

```diff
@@ -18,6 +19,15 @@ const (
 	ModerationText001    = "text-moderation-001"
```

Comment:
@MunaerYesiyan I would like you to add a comment indicating that this constant is deprecated.

Reply:
Deprecating the `text-moderation-001` model in `moderation.go` https://github.com/sashabaranov/go-openai/pull/437/commits/e579559797ea9199d0bd902248c58830a4a23283